athenahealth/patients.go

```diff
@@ -46,15 +46,15 @@ type ListPatientsOptions struct {
 	LastName  string
 }
 
-type listPatientsResponse struct {
+type ListPatientsResponse struct {
```

Comment:
This struct is only used within the athenahealth package, so it doesn't need to be exported.